extensions/wikia/Search/classes/Result/ResultHelper.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace Wikia\Search\Result;
+
+use Wikia\Search\MediaWikiService, Wikia\Search\Utilities, CommunityDataService, ImagesService, WikiaSearchController, PromoImage;
+
+class ResultHelper
+{
+	const MAX_WORD_COUNT_EXACT_MATCH = 16;
+	const MAX_WORD_COUNT_XWIKI_RESULT = 60;
+
+	/**
+	 * +     * Extends search result with additional data from outside search index, like description and image
+	 * +     *
+	 * +     * @param $result
+	 * +     * @param $pos
+	 * +     * @param $descWordLimit
+	 * +     * @return array
+	 * +     */
+	public static function extendResult($result, $pos, $descWordLimit)
+	{
+
+		$thumbTracking = 'class="wiki-thumb-tracking" data-pos="' . $pos . '" data-event="search_click_wiki-thumb"';
```

Comment:
Is it possible (reasonable) to place those classes and data attributes in template?